_dashboards/index.md

```diff
@@ -22,6 +24,8 @@ You can try OpenSearch Dashboards' features in the [OpenSearch Dashboards Playgr
 
 Choose **OpenSearch Dashboards > Dashboards > Create dashboard** and explore the sample Global Flight, Web Traffic, and Revenue dashboards.
 
+![Dashboards playground screenshot]({site.url}}{{site.baseurl}}/images/dashboards-playground.png)
```

Comment:
Missed a curly bracket at the beginning of `site.url` - probably just a copy-paste error but make sure to add it otherwise the link won't display correctly.